Mathlib/Data/Finset/Lattice.lean

```diff
@@ -1135,6 +1143,12 @@ theorem inf'_mono {s₁ s₂ : Finset β} (h : s₁ ⊆ s₂) (h₁ : s₁.Nonem
     s₂.inf' (h₁.mono h) f ≤ s₁.inf' h₁ f :=
   Finset.le_inf' h₁ _ (fun _ hb => inf'_le _ (h hb))
 
+/-- A version of `Finset.inf'_mono` acceptable for `@[gcongr]`. -/
```

Comment:
I don't remember what I wrote here precisely, probably it was asking for slightly more explanation which changes were needed: is it just that `h₂` is added or also that `h₁` and `h₂` are marked as implicit now?